pkg/micropython/Makefile

```diff
@@ -0,0 +1,14 @@
+PKG_NAME=micropython
+PKG_URL=https://github.com/kaspar030/micropython
```

Comment:
Could we have this pulled directly from upstream with our patches/contrib added on top?

Currently to know what has been changed I have to go to your fork and diff it against upstream.

Reply:
> Could we have this pulled directly from upstream with our patches/contrib added on top?

Nope, that is a maintanance nightmare.

> Currently to know what has been changed I have to go to your fork and diff it against upstream.

Yes. ```make; cd bin/pkg/native/micropython; git diff master```.

Compare that to the extra work necessary to change only a single line in the micropython fork...